message/router.go

```diff
@@ -399,17 +443,20 @@ type handler struct {
 	closeCh chan struct{}
 }
 
-func (h *handler) run(middlewares []HandlerMiddleware) {
+func (h *handler) run(middlewares []Middleware) {
 	h.logger.Info("Starting handler", watermill.LogFields{
 		"subscriber_name": h.name,
 		"topic":           h.subscribeTopic,
 	})
 
 	middlewareHandler := h.handlerFunc
-
-	// first added middlewares should be executed first (so should be at the top of call stack)
-	for i := len(middlewares) - 1; i >= 0; i-- {
-		middlewareHandler = middlewares[i](middlewareHandler)
+	for i := 0; i < len(middlewares); i++ {
```

Comment:
so as we discussed - it should be done like before ;) (middlewares are applied in reversed order because the "outside" middleware is called first)

Reply:
Yes, you are right I will remove the change.